AC_FIND_XFT looks for the fontconfig-config program which does not
exist any more. Fall back to "pkg-config fontconfig".
https://bugs.gentoo.org/462232
http://bugs.motifzone.net/show_bug.cgi?id=1593

--- motif-2.3.4-orig/ac_find_xft.m4
+++ motif-2.3.4/ac_find_xft.m4
@@ -189,7 +189,7 @@
 yes)
 	case "$fc_config" in
 	no)
-		FONTCONFIG_CFLAGS=""
+		FONTCONFIG_CFLAGS="`pkg-config fontconfig --cflags`"
 		;;
 	*)
 		FONTCONFIG_CFLAGS="`$fc_config --cflags`"
@@ -208,7 +208,7 @@
 yes)
 	case "$fc_config" in
 	no)
-		fontconfig_lib=""
+		FONTCONFIG_LIBS="`pkg-config fontconfig --libs`"
 		;;
 	*)
 		FONTCONFIG_LIBS="`$fc_config --libs`"
